src/nginx/Chart.yaml

```diff
@@ -1,17 +1,5 @@
-appVersion: 1.10.2
-description: Chart for the nginx server
-engine: gotpl
-home: http://www.nginx.org
-keywords:
-- nginx
-- http
-- web
-- www
-- reverse proxy
-maintainers:
-- email: containers@bitnami.com
-  name: Bitnami
+apiVersion: v1
```

Comment:
Why is most of this file removed?

Reply:
IMO this is a new chart, it is created from a "helm create" and supports the same features.